implement raw_solr_query - closes #7

phated opened the pull request:
Still needs tests, but am I on the right track with this?

phated committed:
implement raw_solr_query - closes #7

dmitrizagidulin commented on lib/superagent_transport.js:
```diff
@@ -42,6 +42,15 @@ SuperagentTransport.prototype.get_text = function get_text(path, callback) {
     });
 }
 
+SuperagentTransport.prototype.get_request = function put_request(url, json_obj, callback) {
```
Is the `prototype.get_request` = `put_request()` mismatch on purpose? And anyways, you can probably just use `transport.get_body()` instead, no?

phated replied:
It was a copy-paste mistake, I will fix that.  I am not yet sure if it makes sense to only get the body back or the whole request, so I implemented it in this fashion.

dmitrizagidulin commented:
Yeah, that's it! 
Just let me know what you meant to do re the get_request() above, and I'll merge it.

phated committed:
fix tests and function name typo

phated committed:
integration test for raw solr query

phated commented:
The `get_request` function was added because the other `get_*` methods don't take query parameters.  I fixed the typo of `put_request`.

I added an integration test for the `raw_solr_query` method and cleaned up some tests + the seed file to wait for the `set_schema` callback before proceeding.  Also, increased the timeout for a slow dev instance.
